cocos/rendering/custom/executor.ts

```diff
@@ -332,6 +333,40 @@ class BlitDesc {
             }
         }
 
+        for (let i = 0; i < pointLights.length && idx < maxLights; i++, ++idx) {
+            const light = pointLights[i];
+            Sphere.set(_sphere, light.position.x, light.position.y, light.position.z, light.range);
+            if (intersect.sphereFrustum(_sphere, camera.frustum)) {
+                // cc_lightPos
+                Vec3.toArray(_vec4Array, light.position);
+                _vec4Array[3] = 0;
+                this._lightBufferData.set(_vec4Array, idx * elementLen);
+
+                // cc_lightColor
+                Vec3.toArray(_vec4Array, light.color);
+                if (light.useColorTemperature) {
+                    const tempRGB = light.colorTemperatureRGB;
+                    _vec4Array[0] *= tempRGB.x;
+                    _vec4Array[1] *= tempRGB.y;
+                    _vec4Array[2] *= tempRGB.z;
+                }
```

Comment:
Can just record data in array in light? Then don't have to convert it every frame.

Reply:
no

color temperature and light color are two separate values and need to be displayed separately in the editor. You cannot pre-multiply light color after filling in color temperature, otherwise the color temperature displayed in the editor will be incorrect

Reply:
Already solved

Reply:
But i still do conversion here. Can record the value of `_vec4Array ` in light.

Reply:
![图片](https://user-images.githubusercontent.com/20809296/219994537-ae4b8b9d-d6f9-49a1-bda5-a414ab1e0a19.png)

is already changed, github doesnt update